wiki/advantages-and-features.md

```diff
@@ -0,0 +1,185 @@
+---
+title: Gridcoin Advantages & Features
+layout: wiki
+---
+
+
+# Competitive Advantages 
+
+### General Advantages
+
+One of the main advantages to Gridcoin is that crunching ("mining")
+does useful scientific research. Crunching helps science in areas like fighting 
+diseases, finding asteroids, understanding subatomic particles, and more. In other
+cryptocurrencies, mining doesn't produce any meaningful results. This means that 
+regardless of the number of Gridcoin you earn, crunching alone achieve something 
+useful.
+
+While there are some other cryptocurrencies that try and reward scientific work,
+they are limited in other various ways (most are heavily centralized among other 
+things) see:
+* [Comparison to CureCoin](#comparison-to-curecoin)
+* [Comparison to OByte](#comparision-to-obyte)
+* [Comparison to Primecoin](#comparison-to-primecoin)
+* [Comparison to Tokens](#comparison-to-tokens)
+
+Additionally, Gridcoin's system of scrapers offers a scalable system that 
+minimizes that amount of centralization. Scrapers get BOINC statistics 
+(which determine rewards) from the projects and then compress them down to only 
+the useful stats --- reducing data sent and stored. The wallet compares the 
+data from each scraper against each other limiting the trust needed. Scrapers are 
+run by several trusted members of the community located around the world. 
+Read more about scrapers on the [Scraper Wiki Page](Scraper "wikilink")
+
+Further, Gridcoin has a voting system to help keep Gridcoin aligned with the
+community and have meaningful discussions. Voting helps reduce conflicts by
+not having unexpected changes and having a way to voice your opinion without 
+needing to fork and split the community.
+
+
+### Comparison to CureCoin
+---
+#### What's CureCoin?
+CureCoin is a cryptocurrency that aims to reward folding@home work
+
+
+#### Why's Gridcoin Different?
```

Comment:
I would prefer "Why is..."

Reply:
Changed it to `Why Gridcoin Is Different`. I think that might be a bit better